test/evergreen.yml

```diff
@@ -154,67 +174,38 @@ functions:
           set -o verbose
           echo '#undef HAVE_FTRUNCATE' >> config/wiredtiger_config.h
     - *make_wiredtiger
-  "compile wiredtiger address sanitizer":
+  "run data validation stress test checkpoint":
+    - *fetch_artifacts
```

Comment:
Shouldn't we just be using `- func: "fetch artifacts"` here? I don't think there's a need to give the function a variable name, it's only used in one spot.

Reply:
Can't do because its in the functions section I believe.

Reply:
Ack, thanks for looking into it.